demos/common/cpp/models/include/models/hpe_model_associative_embedding.h

```diff
@@ -0,0 +1,59 @@
+/*
+// Copyright (C) 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+#pragma once
+#include "model_base.h"
+
+class HpeAssociativeEmbedding : public ModelBase {
+public:
+    /// Constructor
+    /// @param modelFileName name of model to load
+    /// @param confidenceThreshold - threshold to eleminate low-confidence poses.
+    /// Any pose with confidence lower than this threshold will be ignored.
```

Comment:
Please extend the comment with other params.

Reply:
Updated.